Tests/MapboxMapsTests/Foundation/Extensions/Core/DownloadStatusTests.swift

```diff
@@ -0,0 +1,89 @@
+import XCTest
+@testable import MapboxMaps
+
+final class DownloadStatusTests: XCTestCase {
+
+    func testSuccessRoundtrip1() throws {
```

Comment:
Is this meant to test the httpResult handling specifically? I'm thinking about whether we could come up with a more descriptive name because it's not clear to me what the focus of this test is.